docs/src/ref_impl/development.md

```diff
@@ -0,0 +1,18 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Development
+
+Ghaf Framework is free software, currently under active development.
+
+Once you are up and running, you can participate in the collaborative development process by building a development build with additional options, like the development username and password that are defined in [authentication module](https://github.com/tiiuae/ghaf/blob/main/modules/development/authentication.nix#L4-L5).
+
+If you set up development SSH keys into [SSH module](https://github.com/tiiuae/ghaf/blob/main/modules/development/ssh.nix#L4), you can use `nixos-rebuild switch` to quickly deploy your configuration changes to the development board over the network using SSH:
+
+    nixos-rebuild --flake .#nvidia-jetson-orin --target-host root@orin-hostname --fast switch
```

Comment:
Proposal `root@<orin-hostname-or-ip-address-in-your-network>`

Reply:
we have called  it `ghaf-host` in the default configs, so just call it that?

Also change `.#nvidia-jetson-orin` to `.#nvidia-jetson-orin-debug` as the orin target no longer exists alone

Reply:
Changed this `nixos-rebuild --flake .#nvidia-jetson-orin --target-host root@orin-hostname --fast switch` to this `nixos-rebuild --flake .#nvidia-jetson-orin-debug --target-host root@ghaf-host --fast switch`.